### PR TITLE
report status details, stdout, stderr

### PR DESCRIPTION
**Pull request by MARIANOCEREDA**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side). Also you should start *your branch* off *our main*.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Description

- Added support for fields `executionStatusDetails`, `stdout `and `stderr` of `CustomScriptStatusMessage` in the `result_function` callback for custom commands.
- Sends a `timestamp` in milliseconds since the epoch using `time.time() * 1000`
- Update pydantic version in regex for the test `test_models.py` to pass.
- This aims to mimic [this change](https://github.com/inorbit-ai/edge-sdk/pull/19) that was done for the `js-edge-sdk`. 

### Short Demo

[Screencast from 10-29-2024 09:16:38 AM.webm](https://github.com/user-attachments/assets/0f8dbe5b-41dd-4d03-a422-fb240b7d43f1)

### CODE REVIEW

File: inorbit_edge/inorbit_pb2.py
Comment (miguelgarcia) [inline inorbit_edge/inorbit_pb2.py:1]: Why are there differences in this file ?

**Review by miguelgarcia — COMMENTED** [review]
just a question, code LGTM

File: inorbit_edge/inorbit_pb2.py
Comment (MARIANOCEREDA) [reply inorbit_edge/inorbit_pb2.py:1]: I recompiled it, but it was not necessary. I just rolled back the changes.

Comment (MARIANOCEREDA) [reply inorbit_edge/inorbit_pb2.py:1]: @miguelgarcia I recompiled it, but it was not necessary. I just rolled back the changes.

**Review by miguelgarcia — APPROVED** [review]

**Pull request merged**